File: exs/mundo_1/javascript/026.js
/*
Desafio 026

Problema: Faça um programa que leia uma frase pelo teclado
          e mostre quantas vezes aparece a letra "A", em que
          posição ela aparece a primeira vez e em que posição
          ela aparece a última vez.

Resolução do problema:
*/

const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados

//Entrada de dados
const frase = input('Digite uma frase: '.toUpperCase())

//Quantidade de letra "a"
console.log(`Quantidade de letra "A" ${frase.match(/a/g)}`)

//# Em que posição ela aparece a primeira vez
console.log(frase.indexOf("a",1)) // Aqui vocẽ tem que colocar a posição da letra

// Em que posição ela aparece a última vez
console.log(frase.indexOf("a", 2)) // Aqui vocẽ tem que colocar a posição da letra

File: exs/mundo_1/golang/026.go
/*
Desafio 026

Problema: Faça um programa que leia uma frase pelo teclado
          e mostre quantas vezes aparece a letra "A", em que
          posição ela aparece a primeira vez e em que posição
          ela aparece a última vez.

Resolução do problema:
*/
package main

import (
	"fmt"
	"strings"
)
var x string

func main() {
	fmt.Print("Digite uma palavra: ")
	fmt.Scanln(&x)
	x = strings.ToLower(x)
	c := strings.Count(x, "a")
	i := strings.Index(x, "a")
	l := strings.LastIndex(x, "a")
	fmt.Printf("Index da primeira letra \"A\": %v\n", i)
	fmt.Printf("Quantidade de letras \"A\": %v\n", c)
	fmt.Printf("Index da última letra \"A\": %v\n", l)
}
